src/components/book.js

```diff
@@ -1,9 +1,19 @@
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/book/bookSlice';
+import { getBookItems, removeBookItem } from '../redux/book/bookSlice';
 
 /* eslint-disable react/prop-types */
-const Book = ({ id, title, author }) => {
+/* eslint-disable camelcase */
```

Comment:
- [ ] Good job so far, but note that it is not a good practice to disable linters in your code because linters force you to write bug-free and clean code. Please remove all areas you disabled linters in your app.